param1: 0.075 #0.5
param2: 0.001
param3: 0.1 #0.1
param4: 0.075 
param5: 0.001
param6: 0.1
param7: 2.0
param8: 1.0
param9: 1.5
param10: 0.0
param11: 0.0
param12: 0.0
